src/main/directory-content.ts

```diff
@@ -10,6 +10,7 @@ import {
   appSettings,
   saveAppSettings
 } from "./settings";
+import { pathStartsWith } from "../common/utils/path-utils";
```

Comment:
For some reason Vite refuses to build an app when this import is written like:
`import { pathStartsWith } from "@common/utils/path-utils";`